android/res/values/strings.xml

```diff
@@ -55,6 +55,11 @@
   <string name="go_to_play_store">Go to Play Store</string>
   <string name="no_bookmarks">No bookmarks!</string>
   <string name="menu_bookmarks_list">Bookmarks</string>
-  <string name="no_section_info">No content headers found</string>
-  <string name="request_storage">To access ZIM files we need access to your storage</string>
+  <string name="no_section_info">No Content Headers Found</string>
+  <string name="request_storage">To access zim files we need access to your storage</string>
+  <string name="history_dialog">Are you sure you want to delete all history?</string>
```

Comment:
But it asks if the user wants to delete **all** history when they only want to delete recent searches.